src/stravalib/protocol.py

```diff
@@ -219,7 +219,8 @@ def _request(
         raw = requester(url, params=params)  # type: ignore[operator]
         # Rate limits are taken from HTTP response headers
         # https://developers.strava.com/docs/rate-limits/
-        self.rate_limiter(raw.headers, method)
+        if "/oauth/token" not in url:
```

Comment:
All oauth endpoints are without rate limit headers and are not throttled. Can you make the exclusion URL more generic, e.g. `/oauth/`? That will allow us to also catch e.g. `/oauth/deauthorize`.